mithril-client-cli/src/commands/cardano_db/download/mod.rs

```diff
@@ -1,4 +1,5 @@
-mod shared_steps;
+pub(crate) mod shared_steps;
```

Comment:
I would rather to move the `shared_steps` module to the parent folder and instead of moving the method in  download `v2` to the verify module I would move them to `shared_steps`.
This would keep the command as only users of the shared api, not producer of them.